Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y software-properties-common
 # install Ruby from source
 #  from source: mainly because of GEM native extensions,
 #  this is the most reliable way to use Ruby no Ubuntu if GEM native extensions are required
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential zlib1g-dev libssl-dev libreadline6-dev libyaml-dev libsqlite3-dev
```

Comment:
When you add a new pre-installed tool, you have to add it to the "system_report" script too!

From the README: 

> If you'd like to add a tool to be pre-installed you can create a Pull Request, adding your changes to the Dockerfile of this repository. Please also add a related test/report to the system_report.sh file, which is used to test & list the pre-installed tools.